rs/tests/consensus/utils/BUILD.bazel

```diff
@@ -31,6 +31,8 @@ rust_library(
         "//rs/types/types",
         "@crate_index//:anyhow",
         "@crate_index//:candid",
+        "@crate_index//:chrono",
+        "@crate_index//:futures",
```

Comment:
Are these dependencies still required in the old place? Same for cargo below

Reply:
I removed them from the concrete target for `consensus_performance` but for `Cargo.toml` i can't because other targets use them. I will try to remove `chrono`